### ISSUE TITLE
Unable to perform update multiple entries

### ISSUE THREAD

**Issue by arnthorsnaer** [balderdashy/sails#3334]
According to the documentation I should be able to supply an array as a find criteria when doing updates.
But when executing:
* Model.update([{id:1},{id:2},{id3}], {new:values}).exec......   

I get the following error: 
* [Error (E_UNKNOWN) Encountered an unexpected error] Details: error: invalid input syntax for integer: "NaN"

Any ideas on this? I can't find an example of this in the documentation, just that the docs say that [{}] is an acceptable find criteria for .update.

**Comment by sailsbot** [comment]
Thanks for posting, @arnthorsnaer.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
